### ISSUE TITLE
Bug: Behavior change from previous versions prior v1.12.0

### ISSUE THREAD

**Issue by mikependon** [mikependon/RepoDB#602]
Reported by the user @eriego, some behavioral changes has happened from the previous version. See below.

In the previous version, this is working.

```csharp
var model = new { Id = 1, ColumnToUpdate = "Value" };
connection.Update("NonKeyedTable", model);
```

In which the table named `NonKeyedTable` is a table that has no key from the database.

The issue also exists on the other methods like `Delete` and `Merge`.

**Comment by mikependon** [comment]
The fix is now deployed and is available at [RepoDb v1.12.3](https://www.nuget.org/packages/RepoDb/1.12.3).

**Comment by mikependon** [comment]
This has been fixed and is now available at [RepoDb v1.12.3](https://www.nuget.org/packages/RepoDb/1.12.3). Closing this incident ticket now.
